mern-ecommerce/controllers/order.js

```diff
@@ -61,3 +61,15 @@ exports.updateOrderStatus = (req, res) => {
     }
   );
 };
+
+exports.deleteOrder = async (req, res) => {
+  try {
+    const order = await Order.findByIdAndRemove(req.params.orderId);
+    if (!order) {
+      return res.status(404).json({ message: "Order does not exist" });
```

Comment:
The error message in the 404 response should be more descriptive. Instead of just "Order does not exist", consider including the order ID that was not found, e.g., `res.status(404).json({ message: `Order with ID ${req.params.orderId} does not exist` });`.